### PR TITLE
Refine Simplified Chinese localization

### PR DESCRIPTION
**Pull request by Jerry23011**
Hi, I refined some of the translations after using Ice Cubes for some time

I would like to consult @nixzhu to see if there's any room for improvements.

### CODE REVIEW

File: IceCubesApp/Resources/Localization/zh-Hans.lproj/Localizable.strings [hunk 1]
```diff
@@ -94,8 +94,8 @@
 "settings.support.three.title" = "🤯 很慷慨的捐赠";
 "settings.support.two.subtitle" = "我很喜欢这杯美味的咖啡 ❤️";
 "settings.support.two.title" = "☕️ 很不错的捐赠";
-"settings.support.four.title" = "👽 Who are you Tip";
-"settings.support.four.subtitle" = "It'll go a long way to keep Ice Cubes running!";
+"settings.support.four.title" = "👽 神秘人的小费";
+"settings.support.four.subtitle" = "让 Ice Cubes 持续下去还有很长的路！";
```
Comment (nixzhu) [inline IceCubesApp/Resources/Localization/zh-Hans.lproj/Localizable.strings:98]: 这将大大有助于 Ice Cubes 的持续运行

File: IceCubesApp/Resources/Localization/zh-Hans.lproj/Localizable.strings [hunk 2]
```diff
@@ -297,7 +297,7 @@
 "status.media.content.show" = "显示内容";
 "status.media.contextmenu.copy" = "拷贝图片";
 "status.media.contextmenu.save" = "保存图片";
-"status.media.contextmenu.share" = "分享图片";
+"status.media.contextmenu.share" = "共享图片";
```
Comment (nixzhu) [inline IceCubesApp/Resources/Localization/zh-Hans.lproj/Localizable.strings:300]: “分享” is common.

File: IceCubesApp/Resources/Localization/zh-Hans.lproj/Localizable.strings [hunk 3]
```diff
@@ -259,20 +259,20 @@
 "status.action.message" = "私信";
 "status.action.pin" = "固定";
 "status.action.post" = "嘟嘟";
-"status.action.quote" = "引用嘟文";
+"status.action.quote" = "引用";
 "status.action.reply" = "回复";
 "status.action.section.your-post" = "你的嘟文";
-"status.action.share" = "分享这个嘟文";
+"status.action.share" = "共享嘟文";
```
Comment (nixzhu) [inline IceCubesApp/Resources/Localization/zh-Hans.lproj/Localizable.strings:265]: “分享” is common.

File: IceCubesApp/Resources/Localization/zh-Hans.lproj/Localizable.strings (hunk 1 above)
Comment (Jerry23011) [reply IceCubesApp/Resources/Localization/zh-Hans.lproj/Localizable.strings:98]: Thank you, I will make you co-author with this commit

File: IceCubesApp/Resources/Localization/zh-Hans.lproj/Localizable.strings (hunk 2 above)
Comment (Jerry23011) [reply IceCubesApp/Resources/Localization/zh-Hans.lproj/Localizable.strings:300]: 分享确实更常见，但是我发现 SwiftUI 用的就是共享，所以这才改过来

File: IceCubesApp/Resources/Localization/zh-Hans.lproj/Localizable.strings (hunk 3 above)
Comment (Jerry23011) [reply IceCubesApp/Resources/Localization/zh-Hans.lproj/Localizable.strings:265]: 还是分享好，改回来了

**Comment by nixzhu** [comment]
LGTM

**Comment by Dimillian** [comment]
Nice 🚀

**Pull request merged**